auth/user_mgt_test.go

```diff
@@ -894,17 +873,6 @@ func TestInvalidUpdateUser(t *testing.T) {
 				},
 			}),
 			`the second factor "phoneNumber" for "invalid" must be a non-empty E.164 standard compliant identifier string`,
-		}, {
```

Comment:
nit: I would prefer to keep the test cases related to display name field being required.